required/graphics/testfiles/utf8-filenames-002.tlg

```diff
@@ -91,6 +91,7 @@ l3backend-dvips.def    ....-..-.. L3 backend support: dvips
 graphicx.sty    ....-..-.. v... Enhanced LaTeX Graphics (DPC,SPQR)
   keyval.sty    ....-..-.. v... key=value parser (DPC)
 graphics.sty    ....-..-.. v... Standard LaTeX Graphics (DPC,SPQR)
+graphics-....-..-...sty
```

Comment:
As I commented in https://github.com/latex3/latex2e/issues/1413#issuecomment-2229488694, the version test of base version (here `\ver@graphics.sty`) is still that of the historic version.

Since in the base file (here `graphics.sty`) `\DeclareRelease` (can be one or more) and `\DeclareCurrentRelease` all came before `\ProvidesPackage`, if a historic release is selected the loading of base file is ended before that `\ProvidesPackage` is invoked. Therefore currently there is no way to get the base version.

I once tried with appending some explanatory short text like ` (rolled back)` to the version text `\ver@graphics.sty`, but found that not very helpful.

What do you think?

Reply:
I think we are getting what we need to know - the effective date of the code being used. Yes, it's a little tricky but we quite deliberately are not loading the full `foo.sty` but are switching to `foo-YYYY-MM-DD.sty`, and that is clear in the `.log`.